Regression: S3Event not longer compiles

When using `S3Event` with only the `s3` feature the project fails to compile (I have tried on both 0.8.1 and 0.8.2) with the error:

```
error[E0425]: cannot find function `deserialize_headers` in this scope
   --> /home/lau/.cargo/registry/src/github.com-1ecc6299db9ec823/aws_lambda_events-0.8.2/src/s3/object_lambda.rs:82:32
    |
82  |       #[serde(deserialize_with = "deserialize_headers", default)]
```

and 
```
error[E0425]: cannot find function `serialize_headers` in this scope
   --> /home/lau/.cargo/registry/src/github.com-1ecc6299db9ec823/aws_lambda_events-0.8.2/src/s3/object_lambda.rs:83:30
    |
83  |       #[serde(serialize_with = "serialize_headers")]
```

This is probably because `headers` is behind feature flag `alb` or `apigw`
https://github.com/calavera/aws-lambda-events/blob/f36fd4a63281d57851182a1073468cfe11cbbb11/src/custom_serde/mod.rs#L14
https://github.com/calavera/aws-lambda-events/blob/f36fd4a63281d57851182a1073468cfe11cbbb11/src/custom_serde/mod.rs#L16

If I enable the ie feature flag `alb` the project compiles, but this should not be necessary.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "aws_lambda_events"
-version = "0.8.2"
+version = "0.8.3"
 description = "AWS Lambda event definitions"
 authors = [
   "Christian Legnitto <christian@legnitto.com>",
@@ -99,10 +99,10 @@ dynamodb = ["streams", "serde_dynamo"]
 ecr_scan = []
 firehose = []
 iam = []
-iot = []
+iot = ["iam"]
 iot_1_click = []
 iot_button = []
-iot_deprecated = []
+iot_deprecated = ["iot"]
 kafka = []
 kinesis = []
 kinesis_analytics = ["kinesis"]

--- a/src/custom_serde/mod.rs
+++ b/src/custom_serde/mod.rs
@@ -11,9 +11,21 @@ pub(crate) mod codebuild_time;
 #[cfg(feature = "codebuild")]
 pub type CodeBuildNumber = f32;
 
-#[cfg(any(feature = "alb", feature = "apigw"))]
+#[cfg(any(
+    feature = "alb",
+    feature = "apigw",
+    feature = "s3",
+    feature = "iot",
+    feature = "lambda_function_urls"
+))]
 mod headers;
-#[cfg(any(feature = "alb", feature = "apigw"))]
+#[cfg(any(
+    feature = "alb",
+    feature = "apigw",
+    feature = "s3",
+    feature = "iot",
+    feature = "lambda_function_urls"
+))]
 pub(crate) use self::headers::*;
 
 #[cfg(feature = "dynamodb")]

